packages/cli/src/commands/backend/buildImage.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import fs from 'fs-extra';
+import { createDistWorkspace } from '../../lib/packager';
+import { paths } from '../../lib/paths';
+import { run } from '../../lib/run';
+
+export default async (imageTag: string) => {
+  const tempDistWorkspace = await createDistWorkspace(['example-backend'], {
+    files: [
+      'package.json',
+      'yarn.lock',
+      'app-config.yaml',
+      { src: paths.resolveTarget('Dockerfile'), dest: 'Dockerfile' },
+    ],
+  });
+
+  console.log(`Dist workspace ready at ${tempDistWorkspace}`);
+
+  await run('docker', ['build', '.', '-t', imageTag], {
+    cwd: tempDistWorkspace,
+  });
+
+  await fs.remove(tempDistWorkspace);
```

Comment:
Maybe effectively put in a try-finally?

Reply:
we were thinking about signal handlers to cleanup, but left that for a later good-first-issue